pajbot/models/user.py

```diff
@@ -426,6 +426,16 @@ def find_by_login(db_session: Session, login: str) -> Optional[User]:
     def find_by_id(db_session: Session, id: str) -> Optional[User]:
         return db_session.query(User).filter_by(id=id).one_or_none()
 
+    @property
+    def name_and_login(self) -> str:
```

Comment:
I'd at least like to debate the naming of this function, maybe `display_verbose()` and remove the `@property`? So that ` {{ user.name_and_login }}` becomes `{{ user.display_verbose() }}` - I think this would be clearer. Am open to comments and other people's thoughts on this though.

Reply:
Ended up with a "happy medium", calling it `name_verbose`, making sure the comment was clear, but keeping it a property so it's consistent with `.name` and `.login` in templates